src/LiveComponent/src/Metadata/LiveComponentMetadataFactory.php

```diff
@@ -130,6 +130,17 @@ private static function propertiesFor(\ReflectionClass $class): iterable
         }
     }
 
+    private function createQueryStringMapping(string $propertyName, LiveProp $liveProp): array
```

Comment:
Is it used elsewhere ?

Reply:
Nope, good catch, I think we could remove this function.